scripts/travis-ci/run-test.sh

```diff
@@ -12,7 +12,10 @@ case "$1" in
         ;;
     *)
         ln -s $MODULE_DIR $DRUPAL_DIR/modules/og
+        composer install
         cd $DRUPAL_DIR
+        php modules/composer_manager/scripts/init.php
```

Comment:
You don't need composer manager for that, you can do this:

```
$ composer dump-autoload
```

But that should also not be necessary, the autoload files that are initially created should normally work out of the box. If it needs to be rebuilt this indicates that something is wrong with the order in which you perform your build steps.